.github/workflows/ci.yml

```diff
@@ -21,9 +21,30 @@ jobs:
           npm test
           npm run build
 
+  e2e:
+    name: Playwright Tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
```

Comment:
minor: since these are fairly intuitive, you can remove the name and just keep the run command